src/utils/errors.rs

```diff
@@ -132,11 +129,11 @@ impl From<serde_json::error::Error> for ApiError {
     }
 }
 
-impl From<rocket_contrib::json::JsonError<'_>> for ApiError {
-    fn from(err: JsonError<'_>) -> Self {
+impl From<rocket::serde::json::Error<'_>> for ApiError {
```

Comment:
We can probably get rid of the full module path import here, as you have it in a `use` statement at the top

Reply:
Yes, but I only did a find replace refactor, and no additional changes